scrunch/datasets.py

```diff
@@ -1025,8 +971,17 @@ def itervalues(self):
             yield Variable(var_tuple, self)
 
     def iterkeys(self):
+        """
+        Yield variable alias, since they are unique
+        """
         for var in self._vars:
-            yield var[1].name
+            yield var[1].alias
+
+    def variable_names(self):
+        """
+        Simply return a list of all variable names in the Dataset
+        """
+        return [var[1].name for var in self._vars]
```

Comment:
The same way, if you have the catalog resource, you can do `return self._catalog.by('name').iterkeys()`